packages/kbn-grid-layout/grid/grid_panel/drag_handle.tsx

```diff
@@ -50,25 +56,46 @@ export const DragHandle = React.forwardRef<
     },
     [interactionStart, gridLayoutStateManager.accessMode$]
   );
+  const onTouchStart = useCallback(
+    (e: UserTouchEvent) => {
+      e.stopPropagation();
+      interactionStart('drag', e);
+    },
+    [interactionStart]
+  );
+  const onTouchEnd = useCallback(
+    (e: UserTouchEvent) => {
+      e.stopPropagation();
+      interactionStart('drop', e);
+    },
+    [interactionStart]
+  );
 
   const setDragHandles = useCallback(
     (dragHandles: Array<HTMLElement | null>) => {
+      if (gridLayoutStateManager.accessMode$.getValue() !== 'EDIT') {
+        return;
+      }
       setDragHandleCount(dragHandles.length);
       dragHandleRefs.current = dragHandles;
 
       for (const handle of dragHandles) {
         if (handle === null) return;
         handle.addEventListener('mousedown', onMouseDown, { passive: true });
+        handle.addEventListener('touchstart', onTouchStart, { passive: false });
+        handle.addEventListener('touchend', onTouchEnd, { passive: true });
```

Comment:
Do we need seperate `onTouchStart` + `onTouchEnd` callbacks defined? It looks like we could modify the existing `onMouseDown` callback a bit and use it for the `touchstart` event listener too, for example?